packages/apify/src/proxy_configuration.ts

```diff
@@ -55,6 +55,8 @@ export interface ProxyConfigurationOptions {
      */
     apifyProxyCountry?: string;
 
+    useApifyProxy?: boolean;
```

Comment:
remove those changes (including the one on line 528), it should not be public, its an internal hack